docs/en/advanced/Performance-Tuning.md

```diff
@@ -234,3 +234,41 @@ When enabled, the same data blocks are available across multiple workers,
 reducing the amount of available storage capacity for unique data.
 Disabling passive caching is important for workloads that have no concept of locality and whose
 dataset is large compared to the capacity of a single Alluxio worker.
+
+### Optimized Commits for Compute Frameworks
+
+Running with optimized commits through Alluxio can provide an order of magnitude improvement in the
+overall runtime of compute jobs.
+
+Computation frameworks that leverage the Hadoop MapReduce committer pattern (ie. Spark, Hive) are
+not optimally designed for interacting with storages that provide slow renames (mainly Object
+Stores). This is most common when using stacks such as Spark on S3 or Hive on Ceph.
```

Comment:
Could we add an example for one of the stack? The example can be a step by step paragraph or a flow chart.
And a comparison between our previous way (`CACHE_THROUGH`) and optimized way (`ASYNC_THROUGH`).
The comparison can help users have a more direct informative understanding of why we introduced those parameters.

Reply:
Updated, thanks.